lib/screens/profile_onboarding_screen.dart

```diff
@@ -21,21 +27,18 @@ class ProfileOnboarding extends StatefulWidget {
 
 class _ProfileOnboardingState extends State<ProfileOnboarding> {
   final GlobalKey<FormState> _formKey = GlobalKey<FormState>();
+  String mediaPath = '';
+  ImagePicker picker = ImagePicker();
+  bool _isUploading = false;
```

Comment:
If you're not using _isUploading, please remove it.

Reply:
@tonynguyen98 

_isUploading is being used.